chart/README.md

```
@@ -22,21 +22,28 @@ Settings | Description | Default |
-| controllerManager.manager.env.copyDirRoot | Directory for which the operator will work from on the host. Some environments may require this to be set to a specific directory. | /tmp |
-| webhooks.enable | Enable the webhook setup in the operator controller. Default is "true" and is required for production. | "true" |
+| controllerManager.nodeAffinity.matchExpressions | advanced node affinity expressions for the controller manager pod. Cannot be used together with `controllerManager.selectors` — choose one. | [] |
+| controllerManager.manager.env.copyDirRoot | Directory for which the operator will work from on the host. Some environments may require this to be set to a specific directory. | /var/lib/skyhook |
+| controllerManager.manager.env.agentLogRoot | Directory the agent will write logs to on the host. Some environments may require this to be set to a specific directory. | /var/log/skyhook |
+| webhook.enable | Enable the webhook setup in the operator controller. Default is "true" and is required for production. | "true" |
 | controllerManager.manager.env.leaderElection | Enable leader election for the operator controller. Default is "true" and is required for production. | "true" |
 | controllerManager.manager.env.logLevel | Log level for the operator controller. If you want more or less logs, change this value to "debug" or "error". | "info" |
 | controllerManager.manager.env.reapplyOnReboot | Reapply the packages on reboot. This is useful for systems that are read-only. | "false" |
 | controllerManager.manager.env.runtimeRequiredTaint | This feature assumes nodes are added to the cluster with `--register-with-taints` kubelet flag. This taint is assume to be all new nodes, and skyhook pods will tolerate this taint, and remove it one the nodes packages are complete. | skyhook.nvidia.com=runtime-required:NoSchedule | 
-| controllerManager.manager.image.repository | Where to get the image from | "ghcr.io/nvidia/nodewright/operator" |
+| controllerManager.manager.image.repository | Where to get the image from | "nvcr.io/nvidia/skyhook/operator" |
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: trailing_only; Actual: leading_and_trailing; Unexpected leading pipe

(MD055, table-pipe-style)